src/Core/src/Test/ResultMockFactory.php

```diff
@@ -89,7 +89,28 @@ public static function create(string $class, array $data = [])
 
         $initializedProperty->setValue($object, true);
         foreach ($data as $propertyName => $propertyValue) {
-            $property = $reflectionClass->getProperty($propertyName);
+            if ($reflectionClass->hasProperty($propertyName)) {
+                // backward compatibility with `UpperCamelCase` naming
```

Comment:
Is this comment correct? Or should it be removed?

Reply:
hmm.. Maybe it can be improved

Reply:
oohhh.... OK, I understand, I was talking about the property's name, while you talked about the user's input...
You're right, your point of view is simpler, applied your first suggestion

Reply:
Sorry for being unclear.